cmd/tenderdash/commands/version.go

```diff
@@ -2,17 +2,33 @@ package commands
 
 import (
 	"fmt"
+	"runtime"
 
+	"github.com/sasha-s/go-deadlock"
 	"github.com/spf13/cobra"
 
 	"github.com/tendermint/tendermint/version"
 )
 
 // VersionCmd ...
-var VersionCmd = &cobra.Command{
-	Use:   "version",
-	Short: "Show version info",
-	Run: func(cmd *cobra.Command, args []string) {
-		fmt.Println(version.TMCoreSemVer)
-	},
-}
+var VersionCmd *cobra.Command = func() *cobra.Command {
+	verbose := false
+	cmd := &cobra.Command{
+		Use:   "version",
+		Short: "Show version info",
+
+		Run: func(cmd *cobra.Command, args []string) {
+			fmt.Println(version.TMCoreSemVer)
```

Comment:
do we need this line? and other `fmt.Println` calls?

Reply:
I asked the question above, because we usually use `fmt.Println` for printing debug information
what about me, then there is `cobra` functionality like `Command.Println`